rails/app/policies/user_policy.rb

```diff
@@ -11,17 +11,15 @@ def index?
   end
 
   def edit?
-    current_user.admin? || (
-      current_user == user
-    )
+    current_user.super_admin? || (current_user.admin? && (current_user.community_id == user.community_id))
```

Comment:
I believe you removed the possibility for a user to edit themselves, which is what `current_user == user` enabled

Reply:
I think it'd also be good to name the concept you're using here and factor out a separate method
```ruby
def community_user_admin?
   current_user.admin? && (current_user.community_id == user.community_id)
end
```

Reply:
Yep, you're right. Updated.